projects/wvr-elements/src/lib/wvr-nav-list/wvr-nav-list.component.ts

```diff
@@ -1,22 +1,19 @@
 import { Component, Input } from '@angular/core';
 import { Alignment } from '../shared/alignment.enum';
-import { wvrThemeHostBindings, wvrThemeInputs } from '../shared/wvr-styles';
-
+/**
+ * The WvrNavList Component contains the aligned and vertical properties that describe the positioning of the ul list.
```

Comment:
The WvrNavList Component presents a navigation list. Elements within this list must be wvr-nav-li elements and can be either links of action elements.

Reply:
Applied correction